runtime/src/main/scala/com/ing/baker/runtime/javadsl/RuntimeEvent.scala

```diff
@@ -24,10 +25,28 @@ case class RuntimeEvent(name: String, providedIngredients: util.Map[String, Valu
 
   def asScala: scaladsl.RuntimeEvent =
     scaladsl.RuntimeEvent(name, providedIngredients.asScala.toMap)
+
+  def copy(name0: String): RuntimeEvent = {
+    new RuntimeEvent(name0, providedIngredients)
+  }
+
+  def copy(providedIngredients0: util.Map[String, Value]): RuntimeEvent = {
+    new RuntimeEvent(name, providedIngredients0)
+  }
 }
 
 object RuntimeEvent {
 
+  def apply(name: String,
+            providedIngredients: util.Map[String, Value]): RuntimeEvent =
+    new RuntimeEvent(name, providedIngredients)
+
+  def apply(name: String): RuntimeEvent =
```

Comment:
I am not sure if these constructors compile as expected in java, we might need to move them inside the class using the `def this(name: String): RuntimeEvent` pattern

Reply:
Done